Foundation/NSDictionary.swift

```diff
@@ -122,7 +122,8 @@ public class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCodin
     }
     
     public override convenience init() {
-        self.init(objects: nil, forKeys: nil, count: 0)
+        // FIXME: This isn't really safe. Either the required initializer handles optional pointers, or it doesn't.
+        self.init(objects: unsafeBitCast(nil as UnsafePointer<AnyObject>?, to: UnsafePointer<AnyObject>.self), forKeys: unsafeBitCast(nil as UnsafePointer<NSObject>?, to: UnsafePointer<NSObject>.self), count: 0)
```

Comment:
I want to note that this is NOT SAFE in the general case. I don't even want to promise that this won't cause problems in optimized builds today.